generators/server/templates/quarkus/src/test/java/package/TestResources.java.ejs

```diff
@@ -21,5 +21,10 @@ package <%=packageName%>;
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.h2.H2DatabaseTestResource;
 
+<%_ if (databaseType !== 'no') { _%>
 @QuarkusTestResource(H2DatabaseTestResource.class)
+<%_ } _%>
+<%_ if (authenticationType === 'oauth2') { _%>
+@QuarkusTestResource(KeycloakServerTestResource.class)
```

Comment:
It'd be cool if we could mock the identity provider so we don't need to spin up a container (like we can with Spring Security).

Reply:
Done via the Mock Oidc Test Resource